admin-site/src/OccasionsList.svelte

```diff
@@ -99,28 +105,34 @@
 
 </script>
 
-    <Array
-      arrayName = {dateSortedOccasions} 
-      emptyArrayMessage = "This happens on occasion. No occasions for this event yet.">
+<Array
+  arrayName = {dateSortedOccasions} 
+  emptyArrayMessage = "This happens on occasion. No occasions for this event yet.">
 
-      {#each  dateSortedOccasions as item (item.id)}
-        <Button on:click={() => occasionButton(item.id)}
-        buttonHtml = '<h3 class="m-0">{item.label}  - {item.label} </h3> <h5>{item.locationCity} - {moment(item.startDateTime).format("LL")} - id:{item.id}</h5>'
-        value = {item.id}/> 
-      {/each}
-     
-    </Array>
+  {#each  dateSortedOccasions as item (item.id)}
+    <Button on:click={() => occasionButton(item.id)}
+    buttonHtml = '<h3 class="m-0">{item.label}  - {item.label} </h3> <h5>{item.locationCity} - {moment(item.startDateTime).format("LL")} - id:{item.id}</h5>'
```

Comment:
Nothing I’d fix now, but this long complex HTML-as-a-string starts to push the limits of your basic Button component. If we were committed strongly to this UX design I’d probably look at creating an OccasionButton component; since we’re not, leave it like this for now.